app/controllers/ems_common.rb

```diff
@@ -899,6 +900,12 @@ def retrieve_amqp_security_protocols
     [[_('Non-SSL'), 'non-ssl']]
   end
 
+  def retrieve_container_security_protocols
+    [[_('SSL'), 'ssl-with-validation'],
+     [_('SSL trusting custom CA'), 'ssl-with-validation-custom-ca'],
+     [_('SSL without validation'), 'ssl-without-validation']]
```

Comment:
@cben @jhernand these seems a little verbose to me. They could be: `'ssl'`, `'ssl-custom-ca'` and `'ssl-insecure'`.
I think it's not required to specify `with-validation` in general because SSL in fact has validation by default. Is the `'ssl'` value clashing with a pre-existing one that implied no validation? Are these values the ones ending up in the db?
Anyway this is just a suggestion, the rest LGTM.